hoot-core/src/main/cpp/hoot/core/elements/OsmMap.h

```diff
@@ -325,6 +330,8 @@ class OsmMap : public std::enable_shared_from_this<OsmMap>, public ElementProvid
 
   std::vector<std::shared_ptr<Roundabout>> _roundabouts;
 
+  std::shared_ptr<IdSwap> _idSwap;
+
```

Comment:
`IdSwap` object acts like the roundabouts, collecting information for later operations.